roles/virtualenv3/tasks/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: make sure we have python3.6
+  pkgng:
+    name: "{{item}}"
+    state: present
+  with_items: 
+    - python36
+    - py27-virtualenv
```

Comment:
`py27-virtualenv`? Should this be `py36-virtualenv`?

Reply:
virtualenv is part of python2. with python3 that is pyenv, and I am not sure how well it is supported by ansible.

Reply:
The `virtualenv_command` comments mention `pyvenv`. It appears you can have `virtualenv_command: "-m venv"` too.

Reply:
I used that:
https://stackoverflow.com/questions/42579168/ansible-create-a-virtualenv-using-the-venv-module